thermal-cam-ctrl/src/pairing.rs

```diff
@@ -102,6 +107,28 @@ impl Pair {
 ///////////////////////////
 // ---- Helper Code ---- //
 ///////////////////////////
+fn power_cycle_heat_camera() -> Result<()> {
+    let orb_os_release =
+        OrbOsRelease::read_blocking().wrap_err("Failed to read /etc/os-release")?;
+
+    match orb_os_release.orb_os_platform_type {
+        OrbOsPlatform::Diamond => {
+            info!("Power-cycling heat camera (2v8 line) using orb-mcu-util (Diamond platform)");
+
+            let status = Command::new("orb-mcu-util")
+                .args(["power-cycle", "heat-camera"])
+                .status()
+                .wrap_err("Failed to execute orb-mcu-util power-cycle heat-camera")?;
+            if !status.success() {
+                return Err(eyre!(
+                    "orb-mcu-util power-cycle heat-camera exited with non-zero status: {status}"));
+            }
+        }
+        OrbOsPlatform::Pearl => {}
```

Comment:
Is it impossible to power cycle the camera on pearl? If its possible, we should power cycle on pearl too.